packages/npm/@amazeelabs/gatsby-source-silverback/test/schema/schema.graphql

```diff
@@ -1,6 +1,9 @@
 scalar Email
 
 type Query {
+  value: String @echo(msg: "value from schema")
+  argument(msg: String!): String @echo(msg: "$msg")
+  parent: String @echo(msg: "parent value") @echo(msg: "$")
```

Comment:
Holy moly 🤑 Black magic 🙈

Reply:
Any sufficiently advanced technology is indistinguishable from magic.